client/ayon_deadline/addon.py

```diff
@@ -67,26 +72,146 @@ def get_deadline_pools(webservice, auth=None, log=None):
             RuntimeError: If deadline webservice is unreachable.
 
         """
+        endpoint = "{}/api/pools?NamesOnly=true".format(webservice)
+        return DeadlineAddon._get_deadline_info(
+            endpoint, auth, log, item_type="pools")
+
+    @staticmethod
+    def get_deadline_groups(webservice, auth=None, log=None):
```

Comment:
I would preffer more explicit name than `webservice`.
```suggestion
    def get_deadline_groups(webservice_url, auth=None, log=None):
```

Reply:
This got lost with last commit.